commands/keplr.js

```diff
@@ -224,6 +239,17 @@ const keplr = {
     return true;
   },
 
+  async getWalletAddress() {
+    await playwright.switchToKeplrWindow();
+    await module.exports.goToHome();
+    const page = await playwright.keplrWindow();
+    await playwright.waitAndClickByText(notificationPageElements.copyAddress);
+    await page.click(notificationPageElements.copyWalletAddressSelector);
```

Comment:
`waitAndClick` should be used instead of calling click directly

Reply:
This is a temp. I'll update it according to the appropriate helper method after adding implementation for selecting chain in our import wallet flow. Btw, you tried it with `waitAndClick`?